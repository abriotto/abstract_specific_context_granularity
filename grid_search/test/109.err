  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  1%|#8                                                                                                                                                                                          | 1/99 [00:00<00:14,  6.69it/s]  2%|###7                                                                                                                                                                                        | 2/99 [00:00<00:14,  6.72it/s]  4%|#######5                                                                                                                                                                                    | 4/99 [00:00<00:09, 10.55it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:09,  9.65it/s]  8%|###############1                                                                                                                                                                            | 8/99 [00:00<00:09,  9.53it/s]  9%|#################                                                                                                                                                                           | 9/99 [00:01<00:10,  8.83it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:01<00:11,  7.85it/s] 11%|####################7                                                                                                                                                                      | 11/99 [00:01<00:11,  7.44it/s] 13%|########################5                                                                                                                                                                  | 13/99 [00:01<00:09,  9.30it/s] 14%|##########################4                                                                                                                                                                | 14/99 [00:01<00:10,  8.40it/s] 16%|##############################2                                                                                                                                                            | 16/99 [00:01<00:08, 10.20it/s] 18%|##################################                                                                                                                                                         | 18/99 [00:01<00:07, 11.19it/s] 20%|#####################################7                                                                                                                                                     | 20/99 [00:02<00:06, 12.05it/s] 22%|#########################################5                                                                                                                                                 | 22/99 [00:02<00:06, 12.67it/s] 24%|#############################################3                                                                                                                                             | 24/99 [00:02<00:06, 11.31it/s] 26%|#################################################1                                                                                                                                         | 26/99 [00:02<00:06, 12.06it/s] 28%|####################################################8                                                                                                                                      | 28/99 [00:02<00:07,  9.83it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:03<00:08,  8.51it/s] 33%|##############################################################3                                                                                                                            | 33/99 [00:03<00:06, 10.21it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:03<00:05, 11.06it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:03<00:06,  9.53it/s] 39%|#########################################################################6                                                                                                                 | 39/99 [00:04<00:07,  8.51it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:04<00:06,  9.51it/s] 43%|#################################################################################2                                                                                                         | 43/99 [00:04<00:06,  8.76it/s] 45%|#####################################################################################                                                                                                      | 45/99 [00:04<00:06,  8.56it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:04<00:06,  8.04it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:05<00:06,  8.02it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:05<00:06,  7.48it/s] 52%|################################################################################################3                                                                                          | 51/99 [00:05<00:05,  8.16it/s] 53%|##################################################################################################2                                                                                        | 52/99 [00:05<00:06,  7.68it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:05<00:06,  7.24it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:06<00:05,  7.61it/s] 57%|#########################################################################################################7                                                                                 | 56/99 [00:06<00:05,  7.23it/s] 59%|#############################################################################################################5                                                                             | 58/99 [00:06<00:05,  7.65it/s] 60%|###############################################################################################################4                                                                           | 59/99 [00:06<00:05,  7.27it/s] 61%|#################################################################################################################3                                                                         | 60/99 [00:06<00:05,  6.97it/s] 63%|#####################################################################################################################1                                                                     | 62/99 [00:06<00:03,  9.27it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:07<00:04,  7.50it/s] 67%|############################################################################################################################6                                                              | 66/99 [00:07<00:04,  7.82it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:07<00:03,  8.61it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:07<00:03,  8.09it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:08<00:03,  7.46it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:08<00:02,  9.08it/s] 74%|#########################################################################################################################################8                                                 | 73/99 [00:08<00:03,  7.99it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:08<00:03,  7.47it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:08<00:03,  7.15it/s] 77%|###############################################################################################################################################5                                           | 76/99 [00:08<00:03,  6.65it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:09<00:03,  6.49it/s] 79%|###################################################################################################################################################3                                       | 78/99 [00:09<00:03,  6.44it/s] 80%|#####################################################################################################################################################2                                     | 79/99 [00:09<00:03,  6.18it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:09<00:03,  5.78it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:09<00:02,  8.46it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:09<00:02,  7.60it/s] 86%|################################################################################################################################################################5                          | 85/99 [00:09<00:01,  9.42it/s] 88%|####################################################################################################################################################################3                      | 87/99 [00:10<00:01,  8.55it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:10<00:01,  7.85it/s] 90%|########################################################################################################################################################################1                  | 89/99 [00:10<00:01,  7.08it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:10<00:01,  6.79it/s] 93%|#############################################################################################################################################################################7             | 92/99 [00:10<00:00,  7.90it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:11<00:00,  7.49it/s] 95%|#################################################################################################################################################################################5         | 94/99 [00:11<00:00,  6.93it/s] 97%|#####################################################################################################################################################################################3     | 96/99 [00:11<00:00,  9.23it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:11<00:00, 10.18it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.44it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  4%|#######5                                                                                                                                                                                    | 1/25 [00:00<00:04,  5.55it/s] 12%|######################5                                                                                                                                                                     | 3/25 [00:00<00:03,  7.18it/s] 16%|##############################                                                                                                                                                              | 4/25 [00:00<00:03,  6.80it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:03,  6.47it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:02,  6.44it/s] 32%|############################################################1                                                                                                                               | 8/25 [00:01<00:01,  9.27it/s] 40%|##########################################################################8                                                                                                                | 10/25 [00:01<00:02,  7.15it/s] 48%|#########################################################################################7                                                                                                 | 12/25 [00:01<00:01,  7.60it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:01<00:01,  8.71it/s] 60%|################################################################################################################2                                                                          | 15/25 [00:01<00:01,  8.22it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:02<00:00,  8.18it/s] 76%|##############################################################################################################################################1                                            | 19/25 [00:02<00:00,  9.27it/s] 84%|#############################################################################################################################################################                              | 21/25 [00:02<00:00, 10.94it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:02<00:00, 13.98it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  9.48it/s]
