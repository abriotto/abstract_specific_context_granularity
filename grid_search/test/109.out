# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6928557753562927, "acc": 0.5288608074188232, "length": 3.4433963298797607, "mode": "train", "epoch": 1}
{"loss": 0.6716190576553345, "acc": 0.5988487601280212, "length": 3.138662338256836, "mode": "test", "epoch": 1}
{"loss": 0.6260089874267578, "acc": 0.6582673788070679, "length": 3.7461190223693848, "mode": "train", "epoch": 2}
{"loss": 0.5688999891281128, "acc": 0.7302611470222473, "length": 4.0, "mode": "test", "epoch": 2}
{"loss": 0.5194704532623291, "acc": 0.7377263307571411, "length": 3.9662253856658936, "mode": "train", "epoch": 3}
{"loss": 0.5103108286857605, "acc": 0.770473062992096, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.47051963210105896, "acc": 0.7823663949966431, "length": 3.9792001247406006, "mode": "train", "epoch": 4}
{"loss": 0.4858670234680176, "acc": 0.7874388694763184, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.4463675618171692, "acc": 0.8040245771408081, "length": 3.988363265991211, "mode": "train", "epoch": 5}
{"loss": 0.45979341864585876, "acc": 0.8018760085105896, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.4205400049686432, "acc": 0.8179793953895569, "length": 3.99164080619812, "mode": "train", "epoch": 6}
{"loss": 0.4461487829685211, "acc": 0.8036705851554871, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.395340234041214, "acc": 0.8293604254722595, "length": 3.995412826538086, "mode": "train", "epoch": 7}
{"loss": 0.4117378890514374, "acc": 0.8262644410133362, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.36842575669288635, "acc": 0.8422644734382629, "length": 3.997504949569702, "mode": "train", "epoch": 8}
{"loss": 0.3962148129940033, "acc": 0.8356444835662842, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.3500821590423584, "acc": 0.8557336330413818, "length": 3.9980978965759277, "mode": "train", "epoch": 9}
{"loss": 0.38308480381965637, "acc": 0.8389068245887756, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.32955124974250793, "acc": 0.8650590777397156, "length": 3.9988443851470947, "mode": "train", "epoch": 10}
{"loss": 0.37356650829315186, "acc": 0.8401305079460144, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.3157042860984802, "acc": 0.8694756627082825, "length": 3.999237060546875, "mode": "train", "epoch": 11}
{"loss": 0.33549222350120544, "acc": 0.8623165488243103, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.2951418459415436, "acc": 0.8790791630744934, "length": 3.999237060546875, "mode": "train", "epoch": 12}
{"loss": 0.3182661235332489, "acc": 0.8725122809410095, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.27575594186782837, "acc": 0.884899377822876, "length": 3.999257802963257, "mode": "train", "epoch": 13}
{"loss": 0.2972505986690521, "acc": 0.8841761946678162, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.2542586326599121, "acc": 0.896705150604248, "length": 3.99910831451416, "mode": "train", "epoch": 14}
{"loss": 0.2761620581150055, "acc": 0.8949428796768188, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.23267288506031036, "acc": 0.910707950592041, "length": 3.9995548725128174, "mode": "train", "epoch": 15}
{"loss": 0.251681387424469, "acc": 0.9079933166503906, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.22267580032348633, "acc": 0.9169806241989136, "length": 3.999241590499878, "mode": "train", "epoch": 16}
{"loss": 0.23809170722961426, "acc": 0.9159054160118103, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.20444977283477783, "acc": 0.9231252074241638, "length": 3.999746799468994, "mode": "train", "epoch": 17}
{"loss": 0.22187380492687225, "acc": 0.9199022650718689, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.1958751231431961, "acc": 0.9295098185539246, "length": 3.9995970726013184, "mode": "train", "epoch": 18}
{"loss": 0.22021780908107758, "acc": 0.9169657230377197, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.1939883679151535, "acc": 0.9301203489303589, "length": 3.99971604347229, "mode": "train", "epoch": 19}
{"loss": 0.20610642433166504, "acc": 0.9310766458511353, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.18417377769947052, "acc": 0.9344895482063293, "length": 3.9993503093719482, "mode": "train", "epoch": 20}
{"loss": 0.20897896587848663, "acc": 0.9265904426574707, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.18583759665489197, "acc": 0.934421956539154, "length": 3.9997947216033936, "mode": "train", "epoch": 21}
{"loss": 0.19525085389614105, "acc": 0.9336868524551392, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.17512942850589752, "acc": 0.9378024935722351, "length": 3.999908447265625, "mode": "train", "epoch": 22}
{"loss": 0.19089584052562714, "acc": 0.9360520839691162, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.17767128348350525, "acc": 0.9344275593757629, "length": 3.999903678894043, "mode": "train", "epoch": 23}
{"loss": 0.19374798238277435, "acc": 0.9348286986351013, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.168927863240242, "acc": 0.9405220746994019, "length": 3.9999136924743652, "mode": "train", "epoch": 24}
{"loss": 0.18685929477214813, "acc": 0.9365416765213013, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.15760071575641632, "acc": 0.946585476398468, "length": 3.99971342086792, "mode": "train", "epoch": 25}
{"loss": 0.18207262456417084, "acc": 0.9377650618553162, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.15967398881912231, "acc": 0.9458869099617004, "length": 3.999929189682007, "mode": "train", "epoch": 26}
{"loss": 0.18389497697353363, "acc": 0.940293550491333, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.16013236343860626, "acc": 0.9432610273361206, "length": 3.9997129440307617, "mode": "train", "epoch": 27}
{"loss": 0.18274304270744324, "acc": 0.9387437105178833, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.15211154520511627, "acc": 0.9485533237457275, "length": 3.9999241828918457, "mode": "train", "epoch": 28}
{"loss": 0.17558318376541138, "acc": 0.9420881867408752, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.14707836508750916, "acc": 0.9523383378982544, "length": 3.9999001026153564, "mode": "train", "epoch": 29}
{"loss": 0.17883391678333282, "acc": 0.9444535970687866, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.14907723665237427, "acc": 0.951384425163269, "length": 3.9999353885650635, "mode": "train", "epoch": 30}
{"loss": 0.1651162952184677, "acc": 0.9473084211349487, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.14367489516735077, "acc": 0.9543254971504211, "length": 3.999957323074341, "mode": "train", "epoch": 31}
{"loss": 0.15239830315113068, "acc": 0.9524470567703247, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.13873380422592163, "acc": 0.954707682132721, "length": 3.9999730587005615, "mode": "train", "epoch": 32}
{"loss": 0.1580176055431366, "acc": 0.9515497088432312, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.13796627521514893, "acc": 0.9568317532539368, "length": 3.99999737739563, "mode": "train", "epoch": 33}
{"loss": 0.15559610724449158, "acc": 0.949999988079071, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.13474878668785095, "acc": 0.9561492204666138, "length": 3.999945878982544, "mode": "train", "epoch": 34}
{"loss": 0.1528754085302353, "acc": 0.9547308087348938, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.12778909504413605, "acc": 0.9590362310409546, "length": 3.9999969005584717, "mode": "train", "epoch": 35}
{"loss": 0.1473466455936432, "acc": 0.9582380056381226, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.12432776391506195, "acc": 0.9594717025756836, "length": 3.999994993209839, "mode": "train", "epoch": 36}
{"loss": 0.14603577554225922, "acc": 0.9579934477806091, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.12195789068937302, "acc": 0.961837649345398, "length": 3.99993896484375, "mode": "train", "epoch": 37}
{"loss": 0.15319941937923431, "acc": 0.9544045329093933, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.11692841351032257, "acc": 0.9633690118789673, "length": 3.999527931213379, "mode": "train", "epoch": 38}
{"loss": 0.14821670949459076, "acc": 0.9549754858016968, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.12071536481380463, "acc": 0.9620574712753296, "length": 3.999992847442627, "mode": "train", "epoch": 39}
{"loss": 0.1622946560382843, "acc": 0.9520391225814819, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.12695424258708954, "acc": 0.9584372639656067, "length": 3.9999935626983643, "mode": "train", "epoch": 40}
{"loss": 0.14008580148220062, "acc": 0.9606037735939026, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.12086191028356552, "acc": 0.9612130522727966, "length": 3.999985694885254, "mode": "train", "epoch": 41}
{"loss": 0.15974512696266174, "acc": 0.9523652791976929, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.13559788465499878, "acc": 0.958056628704071, "length": 3.999999761581421, "mode": "train", "epoch": 42}
{"loss": 0.16311736404895782, "acc": 0.9528547525405884, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.12844513356685638, "acc": 0.9591720104217529, "length": 3.999986171722412, "mode": "train", "epoch": 43}
{"loss": 0.1573651134967804, "acc": 0.9526102542877197, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.1168508306145668, "acc": 0.9636362195014954, "length": 3.999997615814209, "mode": "train", "epoch": 44}
{"loss": 0.14490050077438354, "acc": 0.9583198428153992, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.10945317149162292, "acc": 0.9666286110877991, "length": 3.9999656677246094, "mode": "train", "epoch": 45}
{"loss": 0.14730104804039001, "acc": 0.9588906764984131, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.10759682208299637, "acc": 0.9670920372009277, "length": 3.9999845027923584, "mode": "train", "epoch": 46}
{"loss": 0.13555331528186798, "acc": 0.9614192843437195, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.104086734354496, "acc": 0.9668206572532654, "length": 3.999995708465576, "mode": "train", "epoch": 47}
{"loss": 0.12688441574573517, "acc": 0.9627241492271423, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.11538371443748474, "acc": 0.963037371635437, "length": 3.999992609024048, "mode": "train", "epoch": 48}
{"loss": 0.14316290616989136, "acc": 0.9575855731964111, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.11816804856061935, "acc": 0.9635819792747498, "length": 3.999999523162842, "mode": "train", "epoch": 49}
{"loss": 0.14898329973220825, "acc": 0.9526100158691406, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.11138749122619629, "acc": 0.9651318788528442, "length": 3.99997878074646, "mode": "train", "epoch": 50}
{"loss": 0.14253930747509003, "acc": 0.9592986106872559, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.11085879057645798, "acc": 0.9661402106285095, "length": 3.999992609024048, "mode": "train", "epoch": 51}
{"loss": 0.15554536879062653, "acc": 0.9490212798118591, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.11538802087306976, "acc": 0.9645182490348816, "length": 3.9996049404144287, "mode": "train", "epoch": 52}
{"loss": 0.14663748443126678, "acc": 0.9548940062522888, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.11471749097108841, "acc": 0.9648883938789368, "length": 3.999999523162842, "mode": "train", "epoch": 53}
{"loss": 0.1444285660982132, "acc": 0.9549755454063416, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.10222219675779343, "acc": 0.9692160487174988, "length": 4.0, "mode": "train", "epoch": 54}
{"loss": 0.14599698781967163, "acc": 0.9584827423095703, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.10092493891716003, "acc": 0.9700866341590881, "length": 3.999992609024048, "mode": "train", "epoch": 55}
{"loss": 0.13693825900554657, "acc": 0.9617455005645752, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.102828748524189, "acc": 0.9684255719184875, "length": 3.999974489212036, "mode": "train", "epoch": 56}
{"loss": 0.1349903643131256, "acc": 0.9606033563613892, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.10218645632266998, "acc": 0.9692157506942749, "length": 3.999992609024048, "mode": "train", "epoch": 57}
{"loss": 0.1401471346616745, "acc": 0.9584828615188599, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.09649989753961563, "acc": 0.9706112742424011, "length": 3.9994726181030273, "mode": "train", "epoch": 58}
{"loss": 0.1337127983570099, "acc": 0.9646003246307373, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.10609561204910278, "acc": 0.9658411145210266, "length": 4.0, "mode": "train", "epoch": 59}
{"loss": 0.1463019996881485, "acc": 0.9587274789810181, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.10401226580142975, "acc": 0.9673107862472534, "length": 4.0, "mode": "train", "epoch": 60}
{"loss": 0.13355541229248047, "acc": 0.9575856328010559, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.102324940264225, "acc": 0.9670659303665161, "length": 3.999998092651367, "mode": "train", "epoch": 61}
{"loss": 0.1283898651599884, "acc": 0.9635397791862488, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.10429716110229492, "acc": 0.9674742221832275, "length": 4.0, "mode": "train", "epoch": 62}
{"loss": 0.14815817773342133, "acc": 0.9568515419960022, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.1113450825214386, "acc": 0.9645889401435852, "length": 4.0, "mode": "train", "epoch": 63}
{"loss": 0.1570301353931427, "acc": 0.9547308087348938, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.10873375833034515, "acc": 0.966875433921814, "length": 3.9999990463256836, "mode": "train", "epoch": 64}
{"loss": 0.1277637481689453, "acc": 0.9643555879592896, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.09743709862232208, "acc": 0.9693520665168762, "length": 3.999999761581421, "mode": "train", "epoch": 65}
{"loss": 0.12872357666492462, "acc": 0.9615824222564697, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.09244311600923538, "acc": 0.9719106554985046, "length": 3.999999523162842, "mode": "train", "epoch": 66}
{"loss": 0.131663978099823, "acc": 0.9620717763900757, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.09991303086280823, "acc": 0.9682089686393738, "length": 4.0, "mode": "train", "epoch": 67}
{"loss": 0.1335657238960266, "acc": 0.9576672911643982, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.09866578131914139, "acc": 0.9698147177696228, "length": 4.0, "mode": "train", "epoch": 68}
{"loss": 0.13796000182628632, "acc": 0.9601957201957703, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.09024026989936829, "acc": 0.9724279642105103, "length": 4.0, "mode": "train", "epoch": 69}
{"loss": 0.1263350397348404, "acc": 0.9628058075904846, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.08862076699733734, "acc": 0.972291886806488, "length": 4.0, "mode": "train", "epoch": 70}
{"loss": 0.11485004425048828, "acc": 0.965171217918396, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.09152568131685257, "acc": 0.9725368618965149, "length": 4.0, "mode": "train", "epoch": 71}
{"loss": 0.14893333613872528, "acc": 0.9589723348617554, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.09557487815618515, "acc": 0.9671201705932617, "length": 4.0, "mode": "train", "epoch": 72}
{"loss": 0.12867309153079987, "acc": 0.9632952809333801, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.08789126574993134, "acc": 0.972509503364563, "length": 3.999999761581421, "mode": "train", "epoch": 73}
{"loss": 0.13156771659851074, "acc": 0.9632951617240906, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.09391096234321594, "acc": 0.9706578850746155, "length": 3.999983072280884, "mode": "train", "epoch": 74}
{"loss": 0.13892333209514618, "acc": 0.9610928297042847, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.09108056873083115, "acc": 0.9724822044372559, "length": 4.0, "mode": "train", "epoch": 75}
{"loss": 0.12728925049304962, "acc": 0.9667209982872009, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.08645877987146378, "acc": 0.9739248752593994, "length": 4.0, "mode": "train", "epoch": 76}
{"loss": 0.12842439115047455, "acc": 0.9648452401161194, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.08556266129016876, "acc": 0.9741970896720886, "length": 4.0, "mode": "train", "epoch": 77}
{"loss": 0.11904864758253098, "acc": 0.9675366878509521, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.09055294096469879, "acc": 0.9724280834197998, "length": 3.999999761581421, "mode": "train", "epoch": 78}
{"loss": 0.13836191594600677, "acc": 0.9589723348617554, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.09547489881515503, "acc": 0.970223069190979, "length": 4.0, "mode": "train", "epoch": 79}
{"loss": 0.13281336426734924, "acc": 0.9652527570724487, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.08501464128494263, "acc": 0.974959135055542, "length": 3.9999990463256836, "mode": "train", "epoch": 80}
{"loss": 0.12861500680446625, "acc": 0.9633769392967224, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.08618389815092087, "acc": 0.9727545976638794, "length": 3.999999761581421, "mode": "train", "epoch": 81}
{"loss": 0.13441191613674164, "acc": 0.9598694443702698, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.08553875982761383, "acc": 0.9733260273933411, "length": 3.999999761581421, "mode": "train", "epoch": 82}
{"loss": 0.13837219774723053, "acc": 0.961500883102417, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.08792691677808762, "acc": 0.9726729989051819, "length": 4.0, "mode": "train", "epoch": 83}
{"loss": 0.14962564408779144, "acc": 0.9577488303184509, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.09134949743747711, "acc": 0.9726184010505676, "length": 4.0, "mode": "train", "epoch": 84}
{"loss": 0.1564871370792389, "acc": 0.9563621282577515, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.10162363201379776, "acc": 0.9686441421508789, "length": 3.9999911785125732, "mode": "train", "epoch": 85}
{"loss": 0.1549420803785324, "acc": 0.9548124074935913, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.09358628839254379, "acc": 0.971638560295105, "length": 3.999999761581421, "mode": "train", "epoch": 86}
{"loss": 0.14570866525173187, "acc": 0.954730749130249, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.0869029238820076, "acc": 0.9722644686698914, "length": 4.0, "mode": "train", "epoch": 87}
{"loss": 0.12138781696557999, "acc": 0.9653342962265015, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.09080768376588821, "acc": 0.9714480638504028, "length": 4.0, "mode": "train", "epoch": 88}
{"loss": 0.14777474105358124, "acc": 0.9591355323791504, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.09179279953241348, "acc": 0.971230149269104, "length": 4.0, "mode": "train", "epoch": 89}
{"loss": 0.13713453710079193, "acc": 0.9572593569755554, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.08951772749423981, "acc": 0.9714753031730652, "length": 3.999999761581421, "mode": "train", "epoch": 90}
{"loss": 0.13559237122535706, "acc": 0.960440456867218, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.0807199776172638, "acc": 0.9755851030349731, "length": 4.0, "mode": "train", "epoch": 91}
{"loss": 0.12619298696517944, "acc": 0.9623981714248657, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.08078837394714355, "acc": 0.9738977551460266, "length": 4.0, "mode": "train", "epoch": 92}
{"loss": 0.16123753786087036, "acc": 0.9486949443817139, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.09039179235696793, "acc": 0.9704410433769226, "length": 4.0, "mode": "train", "epoch": 93}
{"loss": 0.13155269622802734, "acc": 0.9603589177131653, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.08933688700199127, "acc": 0.9718291163444519, "length": 4.0, "mode": "train", "epoch": 94}
{"loss": 0.1487000584602356, "acc": 0.9548124074935913, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.08643962442874908, "acc": 0.9724006056785583, "length": 3.999999761581421, "mode": "train", "epoch": 95}
{"loss": 0.1305139809846878, "acc": 0.9615824222564697, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.08590830117464066, "acc": 0.9727816581726074, "length": 3.999999761581421, "mode": "train", "epoch": 96}
{"loss": 0.13185957074165344, "acc": 0.9626427292823792, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.08272483944892883, "acc": 0.9744964838027954, "length": 3.999999761581421, "mode": "train", "epoch": 97}
{"loss": 0.16465544700622559, "acc": 0.9576672911643982, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.09087644517421722, "acc": 0.9727544784545898, "length": 3.999999761581421, "mode": "train", "epoch": 98}
{"loss": 0.15384863317012787, "acc": 0.9591352939605713, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.08760969340801239, "acc": 0.9732444882392883, "length": 4.0, "mode": "train", "epoch": 99}
{"loss": 0.1572769433259964, "acc": 0.9587275981903076, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.0858592689037323, "acc": 0.974523663520813, "length": 4.0, "mode": "train", "epoch": 100}
{"loss": 0.16166387498378754, "acc": 0.956280529499054, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7470000982284546
